### PR TITLE
Add stylecop and fx cop as build analyzers

### PR DESCRIPTION
**Pull request by Fuzzylog1cza**
This introduces a form of linting at build time for our code.

People using resharper can include the stylecop extension. The VS Code guys will get the warnings at build time.

### CODE REVIEW

File: FiveRingsDb/FiveRingsDb.csproj [hunk 1]
```diff
@@ -11,6 +12,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2">
```
Comment (programming-wolf) [inline FiveRingsDb/FiveRingsDb.csproj]: Is this part not necessary for the test project, too?

Comment (Fuzzylog1cza) [reply FiveRingsDb/FiveRingsDb.csproj]: It was giving me the same warnings without. I'd prefer cleaning these subnodes out of the primary project actually.

Thoughts?

Comment (programming-wolf) [reply FiveRingsDb/FiveRingsDb.csproj]: Well, if it ain't necessary, remove it :)

Comment (Fuzzylog1cza) [reply FiveRingsDb/FiveRingsDb.csproj]: Done.

**Pull request merged**